core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = { workspace = true }
 codecov = { repository = "solana-labs/solana", branch = "master", service = "github" }
 
 [dependencies]
+ahash = { workspace = true }
```

Comment:
is this the only change in this PR? Code somewhere already using ahash?

Reply:
I think I somehow lost a commit 🤔
I'll move to draft and fix it